### DISCUSSION TITLE
results of WP2_LOH1

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#607] [Q&A] [answered]
when I run WP2_LOH1, I get this result
![image](https://user-images.githubusercontent.com/104721358/178099021-413552a9-a66a-4651-84d1-44e453059ae5.png)

 Please tell me how to get this result as the picture  shows below，and how to generate a `point source`, Is there any reference please?  thank you.
![image](https://user-images.githubusercontent.com/104721358/178099122-bd89f8b6-049d-40c3-807c-12071f29b920.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
I think you should find the answer in one of the following links:
https://seissol.readthedocs.io/en/latest/search.html?q=point+source&check_keywords=yes&area=default
The figure is done with matplotlib, but the script is not provided.
The reference solution (orange) is probably provided by SISMOWINE.
The blue is generated reading the data in one of the receiver files.

**Reply by daisy20170101** [reply]
Hi, here is the link for the reference solution http://www.sismowine.org/index.html?page=compare&mset=WP
on bottom line ref.zip

**Reply by smallriver666** [reply]
> Hi, here is the link for the reference solution http://www.sismowine.org/index.html?page=compare&mset=WP on bottom line ref.zip

Thank you very much for your kind help.
